index.html

```diff
@@ -5,45 +5,123 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
     <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">    <title>Marco's Portfolio</title>
     <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     <link rel="icon" type="image/x-icon" href="images/favicon.png">
 </head>
 <body>
     <header class="header">
         <a href="#" class="logo"><img src="images/logo.svg"></a>
-        <nav>
+        <nav class="burger">
             <a href="#" class="hamburger"><img src="images/hamburger.svg"/></a>
         </nav>
+        <nav >
+            <ul class="menu">
+              <li class="menu-item"><a href="#description">Hello</a></li>
+              <li class="menu-item"><a href="#projects">Portfolio</a></li>
+              <li class="menu-item"><a href="#about">About</a></li>
+              <li class="menu-item"><a href="#form">Contact</a></li>
+            </ul>
+          </nav>
     </header>
 
+    <span class="hr-mobile">
+
```

Comment:
- [ ] Please use an indentation of two spaces consistently in all your documents. THis is a convention and also best practice.